components/Navbar/Navbar.tsx

```diff
@@ -59,68 +60,80 @@ export const Navbar = ({ active }: NavProps) => {
             mode && 'text-white'
           }`}
         >
-          <Link href={'/'}>
-            <a
-              href=""
-              className={`text-[20px] h-full ${
-                mode ? 'hover:bg-gray-800' : 'hover:bg-gray-100'
-              } items-center flex px-2  font-normal ${
-                active === 'home' && 'active'
-              }`}
-            >
-              Home
-            </a>
-          </Link>
-          <Link href={'/projects'}>
-            <a
-              href=""
-              className={`text-[20px] ${
-                mode ? 'hover:bg-gray-800' : 'hover:bg-gray-100'
-              } h-full flex items-center px-2 font-normal ${
-                active === 'projects' && 'active'
-              }`}
-            >
-              Projects
-            </a>
-          </Link>
-          <Link href={'/about'}>
-            <a
-              href="#"
-              className={`text-[20px] ${
-                mode ? 'hover:bg-gray-800' : 'hover:bg-gray-100'
-              } h-full flex items-center px-2 font-normal ${
-                active === 'about' && 'active'
-              }`}
-            >
-              About
-            </a>
-          </Link>
+          <Tooltip content="Home" placement="left" wrapperClassName="h-full">
+            <Link href={'/'}>
+              <a
+                href=""
+                className={`text-[20px] h-full ${
+                  mode ? 'hover:bg-gray-800' : 'hover:bg-gray-100'
+                } items-center flex px-2  font-normal ${
+                  active === 'home' && 'active'
+                }`}
+              >
+                Home
+              </a>
+            </Link>
+          </Tooltip>
+          <Tooltip
+            content="Projects"
+            placement="down"
+            wrapperClassName="h-full"
+          >
```

Comment:
please remove tooltips from Home, Projects and About from the navbar.

Reply:
Done!